include/dlaf/matrix/distribution.h

```diff
@@ -85,6 +130,10 @@ class Distribution {
     return !operator==(rhs);
   }
 
+  const GlobalElementIndex& offset() const noexcept {
```

Comment:
```suggestion
  const TileElementIndex& offset() const noexcept {
```
If I remember correctly our discussion, from any combination of offsets we set the source rank index such that the offset is in a single tile.

Reply:
This may be what you meant, but just to avoid confusion I want to be precise. Currently the offset is normalized to be an offset within the first _block_ which as you know is not necessarily the same as tile offset. I do this because the source rank index and distribution in general only concerns itself with blocks, not tiles. So technically, it should be
```suggestion
  const BlockElementIndex& offset() const noexcept {
```
Do we want to add an alias like that? Or should we treat `TileElementIndex` as something that can be used to index both into tiles and blocks? Alternatively, if `offset` is to return a `TileElementIndex` it should be renamed to `tileElementOffset` and we need a separate `(Global?)TileIndex tileOffset` member.

Reply:
From my naive point of view, I would suggest adding an alias even if it is just for a few places. IMO there is no reason for not adding an alias and ending up re-using a similar one that might create semantic confusion (not a suggestion for now, but for the next future).